.devcontainer/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM docker.io/library/fedora:37
+FROM docker.io/library/fedora:39
```

Comment:
About 40? cc @vstinner @hroncok

Reply:
I'd use 40. It was pushed to docker.io 17 hours ago which makes it the final release of Fedora 40 (scheduled for today).

Reply:
Yeah, in that case, I prefer to update it to 40 if possible. 
(By the way, I am the Fedora fan these days, thanks for all works :))